post-processor/digitalocean-import/post-processor.go

```diff
@@ -260,6 +242,37 @@ func (p *PostProcessor) PostProcess(ctx context.Context, ui packer.Ui, artifact
 	return artifact, false, false, nil
 }
 
+func extractImageArtifact(artifacts []string) (string, error) {
+	var source string
+	artifactCount := len(artifacts)
+
+	if artifactCount == 0 {
+		return "", fmt.Errorf("no artifacts were provided")
+	}
+
+	if artifactCount == 1 {
+		return artifacts[0], nil
+	}
+
+	validSuffix := []string{"raw", "img", "qcow2", "vhdx", "vdi", "vmdk", "tar.bz2", "tar.xz", "tar.gz"}
+
+artifactLoop:
+	for _, path := range artifacts {
+		for _, suffix := range validSuffix {
+			if strings.HasSuffix(path, suffix) {
+				source = path
+				break artifactLoop
+			}
+		}
+	}
```

Comment:
here we could use a return instead of labels, which I think are to be avoided:
```suggestion
	for _, path := range artifacts {
		for _, suffix := range validSuffix {
			if strings.HasSuffix(path, suffix) {
				return path
			}
		}
	}
```